### DISCUSSION TITLE
Collaborators denied acheivements: Starstruck and Arctic Code Vault Contributor

### DISCUSSION THREAD

**Discussion by dvklopfenstein** [community/community#21085] [Other Feature Feedback, Questions, & Ideas]
I was trusted to be a collaborator on https://github.com/tanghaibao/goatools and became the [#1 contributor](https://github.com/tanghaibao/goatools/graphs/contributors), but my work on this project is unrecognized in these acheivements:
* Starstruck
* Arctic Code Vault Contributor

Could you please recognize the contributions of collaborators by including the results of their work in your achievements?

**Comment by sheilalarmour** [comment]
Yes!

Sent from Mail<https://go.microsoft.com/fwlink/?LinkId=550986> for Windows

From: DV Klopfenstein, ***@***.***>
Sent: Friday, July 22, 2022 12:25 PM
To: ***@***.***>
Cc: ***@***.***>
Subject: [GitHub Community] Collaborators denied acheivements: Starstruck and Arctic Code Vault Contributor (Discussion #21085)


I was trusted to be a collaborator on https://github.com/tanghaibao/goatools and became the #1 contributor<https://github.com/tanghaibao/goatools/graphs/contributors>, but my work on this project is unrecognized in these acheivements:

  *   Starstruck
  *   Arctic Code Vault Contributor

Could you please recognize the contributions of collaborators by including the results of their work in your achievements?
